src/main/java/net/oauth/jsontoken/AbstractJsonTokenParser.java

```diff
@@ -0,0 +1,229 @@
+/**
+ * Copyright 2010 Google Inc.
```

Comment:
2020, I guess? I dunno what the rules are.

Reply:
Will fix for all the copyright